test/y2add_on/clients/add-on_auto_test.rb

```diff
@@ -8,7 +8,9 @@
 describe Yast::AddOnAutoClient do
   describe "#import" do
     let(:params) do
-      { "add_on_products" => add_on_products }
+      { "add_on_products" => add_on_products,
```

Comment:
I guess Rubocop would complain here about the formatting, but as we have not enabled it in this package, it's OK. :wink: